src/modules/settings/components/drawer/index.tsx

```diff
@@ -50,6 +50,8 @@ const SettingsDrawer = ({ ...props }: SettingsDrawerProps) => {
   } = useSignIn();
   const { userId } = useAuthStore();
 
+  const isNameInputInvalid = (form.watch('name')?.length ?? 0) <= 2;
```

Comment:
try use this validation by yup...
if you use by yup, i dont need implement this logics on formHelperText